clap_mangen/src/render.rs

```diff
@@ -238,3 +252,41 @@ fn option_default_values(opt: &clap::Arg) -> Option<String> {
 
     None
 }
+
+/// Generates a Vector of Inline Commands to push to the roff
+/// to properly format possible values that an option can take.
+fn format_possible_values(values: Vec<&clap::builder::PossibleValue>) -> Vec<Inline> {
+    let mut formats: Vec<Inline> = vec![];
+    // With Help
+    if values.iter().any(|p| p.get_help().is_some()) {
+        formats.push(Inline::LineBreak);
+        formats.push(roman("Possible values:"));
+        formats.push(Inline::LineBreak);
+        for value in values {
+            formats.push(roman("  - "));
```

Comment:
Should we use roff's support for bulleted lists?

e.g. https://stackoverflow.com/questions/53936179/nested-bulleted-lists-in-roff-groff

Reply:
So I did make those changes, however after extensive reading and testing, there appears to be no way to do this without having a paragraph break between each possible value. I've pushed a commit with those changes, but if you want to revert back i can make that work as well.

```
OPTIONS
       --test=TEST
              [possible values: true, false]

       --color=COLOR
              Some help on the color argument

              Possible values:

                     • green: This has help

                     • blue

       -h, --help
              Print help information
```